index.html

```diff
@@ -745,19 +745,30 @@ <h2>
         A <dfn>navigation scope</dfn> is a [[!URL]] that represents the set of
         URLs to which an <a>application context</a> can be navigated while the
         manifest is <a>applied</a>. The <a>navigation scope</a> of a manifest
-        <var>manifest</var> is <var>manifest</var>["<a>scope</a>"]. If this is
-        <code>undefined</code> (due to an error, or because it was not declared
-        in the manifest), the navigation scope is <dfn>unbounded</dfn>.
+        <var>manifest</var> is <var>manifest</var>["<a>scope</a>"].
       </p>
+      <div class="note" data-link-for="WebAppManifest">
+        <p>
+          If the <a>scope</a> member is not present in the manifest, it
+          defaults to the parent path of the <a>start_url</a> member. For
+          example, if <a>start_url</a> is <code>/pages/welcome.html</code>, and
+          <a>scope</a> is missing, the navigation scope will be
+          <code>/pages/</code> on the same origin. If <a>start_url</a> is
+          <code>/pages/</code> (the trailing slash is important!), the
+          navigation scope will be <code>/pages/</code>.
+        </p>
+        <p>
+          Developers should take care, if they rely on the default behaviour,
+          that all of the application's page URLs begin with the parent path of
```

Comment:
Even better if you could like to the "parent path" algorithm here

Reply:
Is this still needed, given the above? Technically, I do, because if you click on "scope" in that sentence, it links to the parent path algorithm defined in [scope member](https://pr-preview.s3.amazonaws.com/mgiuca/manifest/pull/647.html#scope-member).